plugins/trivy/pkg/image/getimages.go

```diff
@@ -17,6 +17,32 @@ import (
 	"github.com/fairwindsops/insights-plugins/plugins/trivy/pkg/models"
 )
 
+var namespaceBlocklist []string
+var namespaceAllowlist []string
+
+func init() {
+	namespaceBlocklist = strings.Split(os.Getenv("NAMESPACE_BLACKLIST"), ",")
+	namespaceBlocklist = append(strings.Split(os.Getenv("NAMESPACE_BLOCKLIST"), ","), namespaceBlocklist...)
+	namespaceAllowlist = strings.Split(os.Getenv("NAMESPACE_ALLOWLIST"), ",")
+}
+
+func namespaceIsBlocked(ns string) bool {
+	for _, namespace := range namespaceBlocklist {
```

Comment:
No need to lower case ns?

Reply:
Nah that's coming straight from the k8s API, so it'll always be lower